### ISSUE TITLE
Y-axis cutter pin

### ISSUE THREAD

**Issue by Erop1928** [moggieuk/Happy-Hare#318]
I encountered a problem: the basic version of the MMU_CUT_TIP macro does not take into account the possibility of installing a cutter on the Y axis.
Now I think that it would be easier in my case to screw up the original macro, or change the axes of the printer. However, it would be cool if it were possible to select the axis along which the cutting movement will occur.
Perhaps, if I don’t give up (by changing the axes), and I manage to implement the macro, I’ll post my version here.

**Comment by moggieuk** [comment]
That's true it isn't a fully featured macro. I'd be happy to consider a PR for functionality this like, but because it doesn't match my setup it is not something I can easily do.

**Comment by ningpj** [comment]
Old issue. y axis cutting supported in current release.
